examples/dt/adoption/rhv/README.md

```diff
@@ -0,0 +1,61 @@
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 22.12.23         |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Undercloud node - Source osp 17               | vm    | 1      |
+| Controller nodes - Source osp 17              | vm    | 3      |
+| Compute nodes    - Source osp 17              | vm    | 2      |
+
+##### Networks in RH OSP
+
+| Name         | Type        |
+| ----         | ----------- |
+| Ctlplane     | VLAN tagged |
+| Internal-api | VLAN tagged |
+| External     | VLAN tagged |
+| Tenant       | VLAN tagged |
+
+## Services, enabled features and configurations
+| Service                                     | configuration                   | Lock-in coverage?  |
+| ------------------------------------------- | ------------------------------- | ------------------ |
+| RabbitMQ                                    | default                         | Must have          |
+| OVN                                         | default                         | Must have          |
+| galera                                      | default                         | Must have          |
+| Keystone                                    | default                         | Must have          |
+| glance                                      | default                         | Must have          |
+| placement                                   | default                         | Must have          |
+| nova                                        | default                         | Must have          |
+| neutron                                     | default                         | Must have          |
+
+## Considerations/Constraints
+
+1. 3 node openshift cluster on RHEV.
+2. Source openstack with HA is pre-deployed on RHEV.
+3. Network protocol - ipv4
+4. Native VLAN for openstack setup shared between source openstack and target openshift environment.
+
+## Testing
+
+| Test framework   | Special configuration  | Test case to report |
+| ---------------- | ---------------------  | :-----------------: |
+| Tempest/Sanity   | Use rhel image         | ReportPortal        |
+| Workload Test    | Use rhel/cirros image  | ReportPortal        |
+
+## Stages
+
+All stages must be executed in the order listed below.  Everything is required unless otherwise indicated.
+
+1. [Install dependencies for the OpenStack K8S operators](stage1)
```

Comment:
Can we point to the right pages?

Reply:
@psathyan which pages are you referring to ?

Reply:
When I preview the file and click on the link of `1. Install dependencies for the OpenStack K8S operators`, I am getting a 404-page not found error.

Reply:
Done.